src/battle/partner/kooper.c

```diff
@@ -553,34 +553,39 @@ EvtScript N(shellToss) = {
             EVT_BREAK_LOOP
         EVT_END_IF
     EVT_END_LOOP
-    EVT_CALL(action_command_hammer_start, 0, 47, 3)
+#if VERSION_PAL
+    // 47 * DT = 39
+    EVT_CALL(action_command_hammer_start, 0, 38, 3)
+#else
+    EVT_CALL(action_command_hammer_start, 0, 47 * DT, 3)
```

Comment:
<img width="456" alt="image" src="https://user-images.githubusercontent.com/737603/227147547-f03bb766-0616-4bba-8153-0920c53ea598.png">

Reply:
`50 * DT - 3` seems a likely candidate, considering the later instances of 50 * DT

Reply:
Sounds reasonable to me!